src/Passes/Source/Rules/Notation/Call.ipp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Llvm/Llvm.hpp"
+#include "Rules/Notation/Notation.hpp"
```

Comment:
Cyclical inclusion between Notation.hpp and Call.ipp. Applicable to a number of headers.

(To me personally the inclusion order tells in which order to read the sources in the unfamiliar code base. Cyclical inclusion somewhat confuses)

Reply:
This should clearly be fixed.